docs/development.md

```diff
@@ -99,7 +103,21 @@ The dev version of the manifests can be generated with
 
 1. [Install the gcloud cli][gcloud_sdk].
 2. Set project: `gcloud config set project YOUR_PROJECT_NAME`.
-3. Build & push dev images: `make docker-push-dev`.
+3. To build images with custom tags, edit [.bazelrc](./.bazelrc) as follows:
+
+   `build --define=MANAGER_IMAGE_TAG=<YOUR_CUSTOM_TAG_HERE>`
```

Comment:
Is it not possible to set this through normal env vars?

Reply:
the docker-* family of make targets will all work with `make docker-* BAZEL_ARGS="--define=MANAGER_IMAGE_TAG=my-manager-test-tag"`

Reply:
yep. You are right. I did this by editing the `bazelrc` file for my local dev. Will update the doc to take these values on the cli

Reply:
Fixed.